.buildkite/scripts/steps/functional/performance_playwright.sh

```diff
@@ -127,10 +23,10 @@ if [ -d "$JOURNEY_SCREENSHOTS_DIR" ]; then
   cd "$KIBANA_DIR"
 fi
 
-echo "--- report/record failed journeys"
-if [ "${failedJourneys[*]}" != "" ]; then
-  buildkite-agent meta-data set "failed-journeys" "$(printf "%s\n" "${failedJourneys[@]}")"
+# echo "--- report/record failed journeys"
```

Comment:
@spalger one thing I'm not sure of is if in this PR I need to do anything special for reporting failed journeys.

Reply:
If any of the processes run by `procRunner.run()` exit with a non-zero exit code they will throw an error, which if it's not being caught will propagate to the CLI runner and cause it to exit with a non-zero exit code. This means that it _probably_ works by default, but the reporting/logging might be a little sub-par. If you import `createFailError()` from `@kbn/dev-cli-errors` then you could throw your own error and fully customize the output logging for the error state.